src/utils/defaults.js

```diff
@@ -23,8 +23,8 @@ function findFile ( file ) {
 	}
 }
 
-function addJsExtensionIfNecessary ( file ) {
-	return findFile( file ) || findFile( file + '.js' );
+function addExtensionIfNecessary ( file ) {
+	return findFile( file ) || findFile( file + '.js' ) || findFile( file + '.mjs' );
```

Comment:
I believe this is backwards actually - if both files `file.mjs` and `file.js` exist, then rollup should prefer the `.mjs` since that presumably that would contain the ESM syntax rollup leverages.

```js
return findFile( file ) || findFile( file + '.mjs' ) || findFile( file + '.js' );
```